### ISSUE TITLE
Plugin won't start

### ISSUE THREAD

**Issue by TheIronMinerLv** [MeGysssTaa/ReflexIssueTracker#58]
## Reflex Bug Report
**Issue type (console error/false positive/bypass/other):**
Console error, plugin won't start

**Do /reflex debug, and paste here the link you've got. Keep empty if on test server:**
Can't do that

**A video proving the bypass/false positive if reporting a bypass/false positive, a link to your server log if reporting a console error, or a link to your Reflex log if reporting a bypass/false positive:**
Basically I put the plugin for first time in my server, but I can't get in the server, it showed this error in console: https://pastebin.com/aBqnfafs

**Additional information (i.e. how to reproduce the issue you are experiencing etc.):**
I'm using Paperspigot 1.8.8, some kind of NCP build and other plugins, which doesn't affect this issue.

**Comment by TheIronMinerLv** [comment]
Another error: https://pastebin.com/M0Fmpz0b

**Comment by MeGysssTaa** [comment]
Fixed in the next update^^.
